awx_collection/roles/bulk_host_create/meta/argument_specs.yml

```diff
@@ -0,0 +1,49 @@
+---
+argument_specs:
+  main:
+    short_description: An Ansible Role to run a list of ad hoc commands on Ansible Controller.
```

Comment:
Not ad hoc commands - bulk_host_create in this scenario. I'm not actually sure if the bulk host create fits the paradigm of what you're trying to do though. There seems to be a lot of repeated content throughout these roles. People have, in the past, worked on a templating system for redundant modules (because Ansible core requires fairly redundant code structure). Something like that could possibly be applied to generate these.

Reply:
Yea that would be nice because yes the readme's are all very similar

Reply:
Updated, as that was a typo